AfricanReligions/events/oldgods_ancestor_worship_events.txt

```diff
@@ -4571,6 +4832,295 @@ character_event = {
+	option = {
+		name = { text = EVTOPTB_TOG_7530 }
+		random = {
+			chance = 50
+			add_trait = humble
+		}
+	}
+}
+
+#Babbaku flavour event
+character_event = {
+	id = TOG.7531
+	border = GFX_event_normal_frame_religion
+	picture = GFX_evt_tribal_shrine
+
+	desc = {
+		trigger = { has_character_modifier = follower_of_babbaku }
+		text = EVTDESC_TOG_7531
+	}
+
+	is_triggered_only = yes
+
+	option = {
+		name = { text = EVTOPTA_TOG_7531 }
+		random_list = {
+			50 = { hidden_tooltip = { character_event = { id = TOG.7532 days = 3 } } }
+			50 = { hidden_tooltip = { character_event = { id = TOG.7533 days = 3 } } }
+		}
+	}
+	option = {
+		name = { text = EVTOPTB_TOG_7531 }
+		prestige = -5
+	}
+}
+
+character_event = {
+	id = TOG.7532
+	border = GFX_event_normal_frame_religion
+	picture = GFX_evt_afr_berber_wrestling
+
+	desc = {
+		text = EVTDESC_TOG_7532
+	}
+
+	is_triggered_only = yes
+
+	option = {
+		name = { text = EVTOPTA_TOG_7532 }
+		remove_trait = is_fat
+		remove_trait = is_malnourished
+		add_trait = robust
+	}
+}
+
+character_event = {
+	id = TOG.7533
+	border = GFX_event_normal_frame_religion
+	picture = GFX_evt_magic_ritual
+
+	desc = {
+		text = EVTDESC_TOG_7533
+	}
+
+	is_triggered_only = yes
+
+	option = {
+		name = { text = EVTOPTA_TOG_7533 }
+		add_trait = blinded
+	}
+}
+
+#Tan Garki flavour event
+character_event = {
+	id = TOG.7534
+	border = GFX_event_normal_frame_religion
+	picture = GFX_evt_battle_scene
+
+	desc = {
+		trigger = { has_character_modifier = follower_of_tan_garki }
+		text = EVTDESC_TOG_7534
+	}
+
+	is_triggered_only = yes
+
+	option = {
+		name = { text = EVTOPTA_TOG_7534 }
+		military_techpoints = 80
+	}
+}
+
+#Yayan Zanzanna flavour event
+character_event = {
+	id = TOG.7535
+	border = GFX_event_normal_frame_religion
+	picture = GFX_evt_sick_character
+
+	desc = {
+		trigger = { 
+			AND = {
+				has_character_modifier = follower_of_yayan_zanzanna
+				OR = {
+					has_disease_trigger = yes
+					trait = incapable
+				}
+			}
+		}
+		text = EVTDESC_TOG_7535
+	}
+
+	is_triggered_only = yes
+
+	option = {
+		name = { text = EVTOPTA_TOG_7535 }
+		gold = -10
+		remove_trait = incapable
+		remove_disease_trait_effect = yes	
+	}
+}
+
+#Tan Dowa flavour event
+character_event = {
+	id = TOG.7536
+	border = GFX_event_normal_frame_religion
+	picture = GFX_evt_mysterious_forest
+
+	desc = {
+		trigger = { has_character_modifier = follower_of_tan_dowa }
+		text = EVTDESC_TOG_7536
+	}
+
+	is_triggered_only = yes
+
+	option = {
+		name = { text = EVTOPTA_TOG_7536 }
+		add_character_modifier = {
+			name = blessing_of_ayu
+			duration = -1
+		}
+	}
+	option = {
+		name = { text = EVTOPTB_TOG_7536 }
+		add_character_modifier = {
+			name = blessing_of_kadda
+			duration = -1
+		}
+	}
+}
+
+#Anti flavour event
+character_event = {
+	id = TOG.7537
+	border = GFX_event_normal_frame_religion
+	picture = GFX_evt_berber_wrestling
+
+	desc = {
+		trigger = { 
+			has_character_modifier = follower_of_anti
+			is_married = yes
```

Comment:
Again, if this one is only for married people, you have to specify a description for unmarried people too.

Reply:
Done the same as for Yayan Zanzanna